src/reducers/entities/admin.test.js

```diff
@@ -186,9 +175,7 @@ describe('reducers.entities.admin', () => {
                     version: '0.0.1',
                     state: PluginState.PLUGIN_STATE_RUNNING,
                     name: 'Plugin 1',
-                    description: 'The plugin.',
-                    is_prepackaged: false,
-                    active: true,
+                    description: 'The plugin.',                    active: true,
```

Comment:
Looks like this line got merged together accidentally?

Reply:
Done! Surprised that the check style didn't catch that 🤔